language-support/hs/bindings/package.yaml

```diff
@@ -0,0 +1,43 @@
+name:                daml-ledger
+version:             0.1.0.0
+description:         Haskell bindings for a DAML Ledger
+license:             Apache-2.0
+author:              The DAML Authors
+maintainer:          nick.chapman@digitalasset.com
+github:              https://github.com/digital-asset/daml
+
+dependencies:
+- async
+- base >= 4.7 && < 5
+- bytestring
+- containers
+- deepseq
+- exceptions
+- extra
+- grpc-haskell
+- proto3-suite
+- proto3-wire
+- retry
+- text
+- time
+- transformers
+- unliftio
+- vector
+
+library:
+  source-dirs:
+  - src
+  - gen/src
+
+  ghc-options:
+  - -Wall
+  - -Wno-name-shadowing
+
+  - -XGeneralizedNewtypeDeriving
```

Comment:
This should also go in `default-extensions`.

Reply:
Did this. Thank you!